script/create-test-appx.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * @file Creates an appx using the built Electron
+ *
+ * ## Usage
+ *
+ * Take ./out/Debug and turn it into an appx
+ * > node ./script/create-test-appx.js
+ *
+ * Take ./out/Release and turn it into an appx
+ * > node ./script/create-test-appx.js ./out/Release
+ *
+ * ## Certificate
+ *
+ * This script will automatically detect whether or not the development
+ * certificate is installed. If it isn't, it will attempt to install it.
+ * If installation of the certificate fails, you likely don't have enough
+ * rights and need to restart the script "as administrator".
+ */
+
+
+const fs = require('fs-extra')
+const path = require('path')
+const childProcess = require('child_process')
+const electronWindowsStore = require('electron-windows-store')
+
+// Parse arguments
+const args = process.argv.slice(2)
+
+// Constants
+const BUILD_PATH = args[2]
+  ? path.join(process.cwd, args[2])
+  : path.join(__dirname, `../../out/DEBUG`)
+const STAGING_PATH = path.join(__dirname, '../../out/AppContainerStaging')
+const OUT_PATH = path.join(__dirname, '../../out/AppContainer')
+const CERT_PATH = path.join(__dirname, 'appcontainer/electrontest.pfx')
+const ASSETS_PATH = path.join(__dirname, 'appcontainer/Assets')
+const MANIFEST_PATH = path.join(__dirname, 'appcontainer/appxmanifest.xml')
+const SHIM_PATH = path.join(__dirname, 'appcontainer/shim')
+const PREAPPX_PATH = path.join(OUT_PATH, 'pre-appx')
+
+/**
+ * Do we need to copy this file over?
+ *
+ * @param {String} name
+ * @returns {boolean}
+ */
+function isFileNeeded (name) {
+  return !/.*\.(?:(pdb)|(ilk)|(exp)|(lib))$/ig.test(name)
+}
+
+/**
+ * Do we need to copy this directory over?
+ *
+ * @param {String} name
+ * @returns {boolean}
+ */
+function isDirNeeded (name) {
+  return name === 'resources' || name === 'locales'
+}
+
+/**
+ * Starts a child process using the provided executable
+ *
+ * @param fileName      - Path to the executable to start
+ * @param args          - Arguments for spawn
+ * @param options       - Options passed to spawn
+ * @returns {Promise}   - A promise that resolves when the
+ *                      process exits
+ */
+function executeChildProcess (fileName, args, options) {
+  return new Promise((resolve, reject) => {
+    const child = childProcess.spawn(fileName, args, options)
+    const data = []
+
+    child.stdout.on('data', (line) => {
+      data.push(line)
+      console.log(line.toString())
+    })
+    child.stderr.on('data', (line) => {
+      data.push(line)
+      console.log(line.toString())
+    })
```

Comment:
_(minor)_ this could be a single local arrow function that both `child.stdout.on` and `child.stderr.on` both invoke